pkg/crd/fielddesc/testdata/accesscontextmanageraccesslevel-spec.golden.yaml

```diff
@@ -1,21 +1,7 @@
-# Copyright 2023 Google LLC
```

Comment:
I have to confess - I am confused by what this file is.  I think it's just a test that effectively duplicates the CRD.  But seems mostly harmless, and I did spot a questionable accessPolicys vs accessPolicies on a ref later on...